src/QueryM/DatumCacheWsp.purs

```diff
@@ -115,10 +118,32 @@ instance DecodeAeson GetDatumsByHashesR where
         r
         [ "DatumsFound", "value" ]
 
+newtype GetTxByHashR = GetTxByHashR Boolean
```

Comment:
I think it might make sense to link to the `getTxById` issue here, since this will need to change when that's implemented